src/test/resources/Features/login.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Testing SWANGLABS login page
+  Background:
+    Given I open the login page
+    And the Username text box is empty
+    And the Password text box is empty
+
+  Scenario: Valid login into the page
+    Given I have user:
```

Comment:
Nu prea are relevanta primul I have - ce te-ai gandit sa faci cu el ?

Reply:
Am cosinderat  have-ul ca o modalitate de declarare, in sensul asta l-am folosit.